src/rwe/GameScene.h

```diff
@@ -231,9 +231,44 @@ namespace rwe
         bool up{false};
         bool down{false};
 
+        bool leftCtrlDown{false};
+        bool rightCtrlDown{false};
         bool leftShiftDown{false};
         bool rightShiftDown{false};
 
+        bool trackingOn{false};
+        UnitId trackedUnitId;
+
+        void StartTrack()
+        {
+            if (selectedUnits.size())
+            {
+                // sort selection by unit id so repeated 'T' keydown cycles through all units in a group consistently
+                std::vector<UnitId> unitIds;
+                for (const auto& u : selectedUnits)
+                {
+                    unitIds.push_back(u);
+                }
+                sort(unitIds.begin(), unitIds.end());
```

Comment:
Good catch, it seems to be using the definition from the <algorithm> header, but I searched the whole project and see no "using std" or "using std::sort"... so that is mysterious and troubling haha. I wonder if Microsoft put the algorithms header into the global namespace! Also noticed the same with the 'find' done just below that